typescript/src/main.ts

```diff
@@ -62,11 +64,17 @@ function appendMetalTableRow(metal: metals.Metal, table: HTMLTableElement) {
   affinityCell.appendChild(affinityInput);
 
   const mDeltaGCell: HTMLTableCellElement = row.insertCell(-1);
+  if (counter % 2 == 0) {
+    mDeltaGCell.classList.add("grouped", "right_spacing_even");
```

Comment:
Can we achieve this without needing a counter? If we just have "right_spacing" we should be able to use css to determine whether it's odd or even and colour it accordingly? I think it's safer that way as the row striping will always be done in the same place.